src/api/app/views/webui2/webui/staging/workflows/_staging_projects_table.html.haml

```diff
@@ -0,0 +1,26 @@
+%table.table.table-striped.table-bordered.table-sm.dt-responsive.w-100#staging-projects-datatable
+  %thead
+    %tr.text-center
+      %th Staging Project
+      %th Requests
+      %th Problems
+      - if display_actions_column
+        %th Actions
+  %tbody
+    - staging_projects.each do |staging_project|
+      %tr
+        %td.project
+          = render partial: 'overall_state', locals: { staging_project: staging_project }
+        %td.requests
+          = render partial: 'packages_list', locals: { staging_project: staging_project }
+        %td
+          = render partial: 'problems', locals: { staging_project: staging_project }
```

Comment:
@krauselukas this affect our PR as they are renaming the name of the variable in our partial